snippets/isBrowser.md

```diff
@@ -0,0 +1,17 @@
+### isBrowser
+
+Determines if the current runtime environment is a browser so that front-end modules can run on the server (Node)
```

Comment:
I would maybe change the text here a little bit, to "Determines if the current runtime environment is a browser, so that the front-end can be rendered on the server." :)